src/middleware/create-handler.ts

```diff
@@ -0,0 +1,16 @@
+import { NextFunction, Request, RequestHandler, Response } from "express";
+import { Model } from "mongoose";
+
```

Comment:
Read about differences about middlewares and controllers
From an architectural point of view, create-getAll-remove-update functions are controllers, so it is better to move them to controllers folder